scripts/check.py

```diff
@@ -160,16 +160,18 @@ def _check_availability(bin_name: str, name: str = None) -> bool:
     if shutil.which(bin_name) is None:
         if name is None:
             name = bin_name
-        print('{} not found.'.format(name))
+        logging.error('%s not found', name)
```

Comment:
Was this (and other related changes) caught by mypy?
If not, please move them into a new patch.

Reply:
No, it wasn't. Have removed the change.